src/hyperion/external_interaction/callbacks/xray_centre/ispyb_callback.py

```diff
@@ -58,9 +58,11 @@ def activity_gated_start(self, doc: dict):
             self.params = GridscanInternalParameters.from_json(json_params)
             self.ispyb = (
                 Store3DGridscanInIspyb(self.ispyb_config, self.params)
+                # XXX Does this parameter even exist any more?
```

Comment:
It looks like it yeah, but it's a property of the FGS device rather than an input parameter